sequences/misc.yaml

```diff
@@ -1608,3 +1611,23 @@ dig:
 		Length: *
 		ZOffset: 511
 		Offset: 0, 0, 24
+
+hellfire:
+	idle:
+		Length: 1
+		Facings: 32
+
+tibstick:
+    Defaults:
```

Comment:
Well actually i copy pasted that from Astor since he made the shp/seq for it and sended it like that to me, haven´t double check, should I update that?